grblas/__init__.py

```diff
@@ -4,7 +4,7 @@
 _init_params = None
 _SPECIAL_ATTRS = ["lib", "ffi", "Matrix", "Vector", "Scalar",
                   "base", "exceptions", "matrix", "ops", "scalar", "vector"
-                  "unary", "binary", "monoid", "semiring"]
+                  "unary", "binary", "monoid", "semiring", "numpyops"]
```

Comment:
Rather than creating a new top-level module, I would rather have numpy functions registered as normal UDFs using nested names.

`grblas.numpyops.unary.sqrt` looks almost identical to `grblas.unary.numpy.sqrt`, but the latter doesn't need anonymous registration or a new top-level module.